.github/workflows/release.yml

```diff
@@ -13,14 +13,16 @@ jobs:
           distribution: 'temurin'
           java-version: '17'
           cache: 'gradle'
+      - name: Set release tag
+        run: |
+          TAG_NAME="1.$(date "+%Y%m%d.%H%M")-$(echo $GITHUB_SHA | cut -c1-7)"
```

Comment:
Sette nye versjoner med `2.`-prefix for å unngå at gamle versjoner dukker opp når gradle/maven søker etter nyeste versjon?

Reply:
Så nå er det max en release om dagen?

Reply:
Nei, full release tag blir `1.YYYYmmdd.HHMM_SHA`, så burde gå fint med mange pr dag